### ISSUE TITLE
Fix parsing of negative thermal_zone temps

### ISSUE THREAD

**Issue by dimalo** [prometheus/procfs#283]
Hi! I have an issue using node_exporter on an Intel NUCi3BEH running Debian Buster.

``` zsh
$ uname -a
Linux nuc 4.19.0-8-amd64 #1 SMP Debian 4.19.98-1 (2020-01-26) x86_64 GNU/Linux
```

Setting up lm-sensors and basically hitting enter till the end I get these sensor values:
```zsh
$ sensors
pch_cannonlake-virtual-0
Adapter: Virtual device
temp1:        +48.0°C

acpitz-acpi-0
Adapter: ACPI interface
temp1:       -263.2°C
temp2:        +27.8°C  (crit = +119.0°C)

coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +46.0°C  (high = +100.0°C, crit = +100.0°C)
Core 0:        +45.0°C  (high = +100.0°C, crit = +100.0°C)
Core 1:        +44.0°C  (high = +100.0°C, crit = +100.0°C)
```

I'm not running on liquid nitrogen :smile:

node_exporter logs give me following error log:

```zsh
level=error ts=2020-04-22T12:01:48.962Z caller=collector.go:161 msg="collector failed" name=thermal_zone duration_seconds=0.000452796 err="strconv.ParseUint: parsing \"-263200\": invalid syntax"
```

It seems like procfs expects only uint [here](https://github.com/prometheus/procfs/blob/master/sysfs/class_thermal.go#L70)
Unfortunately I'm not familier with go so any pro can help out with this?

Surely this fix would be very interesting for the overclocking community or enabling this software for quantum cores or well, just faulty configured NUC devices :smile:
Thanks!

**Comment by binjip978** [comment]
hello, would like to help with this one, opened #287

**Comment by pgier** [comment]
Fixed by #287
